js/containers/portal-dashboard/portal-dashboard-app.tsx

```diff
@@ -204,7 +204,7 @@ class PortalDashboardApp extends React.PureComponent<IProps, IState> {
 
   private handleScroll = (e: React.UIEvent<HTMLElement>) => {
     const element = e.target as HTMLElement;
-    if (element && element.scrollLeft) {
+    if (element && element.scrollLeft !== undefined) {
```

Comment:
```suggestion
    if (element && element.scrollLeft != null) {
```
It's a matter of style/opinion, but I prefer the simpler (to me) test which captures both null and undefined. Otherwise, I'm left wondering what would happen if the value were ever null.

Reply:
Yes, that's better.  I was bit by this test already so I don't it to come back and bite me again.

Reply:
Fixed.